### ISSUE TITLE
Overload SetText Method

### ISSUE THREAD

**Issue by psuong** [psuong/Unity-UI-Framework#15]
The 

```
SetText(string textName, string screenText, float textSize = 25) 
```

is good and all, but it also get's pretty annoying that you have to set the font size by code. You should make it such that there's also a lazy implementation and that the user can just control text size through the inspector.

**Comment by psuong** [comment]
The Text Component in Unity already has control over text size, you just need to overload SetText to accept two parameters.

```
SetText(string textName, string screenText)
```

Your recent commit is pretty solid though, I can merge with it later tonight.

**Comment by psuong** [comment]
Issue resolved, please see [ddb1365265aebfbca8d78a2d9ab29289d10a9a83](https://github.com/psuong/Unity-UI-Framework/commit/ddb1365265aebfbca8d78a2d9ab29289d10a9a83)
